content/articles/Website-vs-Web-app/index.md

```diff
@@ -0,0 +1,147 @@
+
+A website is a collection of interconnected pages that may contain images, audio, or video and are linked by a single domain name.
+
+Website is of different types. Some of them are:
+
+1. Dating websites
+
+2. Educational Websites
+
+3. Social Media websites
+
+4. Portfolio website
+
+#### Characteristics of A website
+
+A good web designer can Judge a site according to its appearance. One of the characteristics that will determine the success of a website is its appearance. Let's have a look at some additional website attributes.
+ 1. ``User friendly``- A website can be operated by almost everybody for it is easy to use and navigate.
+
+2. ``Attractive and easy for all targeted users`` - This fixture helps in attracting all users and gives them a reason for not choosing another website to visit.
+
+3. ``Purpose``- What is the objective of the website's appearance? A good website should provide a purpose and that purpose should be clear to the user within a short period. A user should not take much of his/her time trying to understand the reason for the appearance of a site.
+
+   The unclear purpose may lead to a website not giving back good results.
+ 
+4. ``High speed of loading``- Web designers should consider this fixture as not all end-users are patient. In this world of time, there is no doubt that time is a very important resource and nobody would like his/her time to go to waste. Therefore if a website loads slowly, end-users likely to look for a different website with a high speed.
+
+    There numerous elements that factor into the loading speed of a website, yet everything starts with the hosting quality. Always choose a host that loads first and keep end-users happy.
+
+5. ``Secure``- A good website provides security for both end-users and designers 
+
+#### What is the importance of a website
+
+1. ``It provides self-service customer support in Business``- If your website support Q&A post, then you can provide a forum where user can have questions about your goods among themselves. Other users can provide answers to the questions.
+
+   This can make things a lot quicker for your clients and make your client service office more effective.
+
+2. ``Website shows Professionalism``- Imagine you are dealing with a supplier whom you want to buy goods from, and this supplier doesn't have a website. I am very sure you would not have a second thought looking for another supplier with a website. This gives an impression that that supplier or company is not well managed.
+
+   Why would an end-user purchase from a supplier or company without a website? Most end-users have a bad opinion of suppliers without a website and they consider them as unprofessional.
+
+3. ``It gives opportunities abroad``- If you have a website, you create an opportunity for everyone worldwide to find you. If you're selling your product online, you end up having more market compared to a business person selling locally. This will in turn increase the profit. 
```

Comment:
I think this can be paraphrased and explained in a better way. Instead talk about creates a large market pool instead of offering opportunities abroad. Remember your article is being read by people worldwide.